app/components/views/Bond/index.tsx

```diff
@@ -749,10 +754,39 @@ export const Bond: FC<Props> = (props) => {
                 <Spinner />
               </div>
             ) : (
-              <ButtonPrimary
-                {...getButtonProps()}
-                className={styles.submitButton}
-              />
+              <Box
+                sx={{ display: "flex", flexDirection: "column", width: "100%" }}
+              >
+                <ButtonPrimary
+                  {...getButtonProps()}
+                  className={styles.submitButton}
+                />
+                {view === "redeem" && !showSpinner && (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      width: "100%",
+                      alignItems: "center",
+                      justifyContent: "center",
+                      paddingRight: "16px",
```

Comment:
We use `rem` everywhere because it scales with the user's font-size. I put an override in the root stylesheet so that 1rem is always 10px.
```suggestion
                      paddingRight: "1.6rem",
```

Reply:
updated, I will keep this in mind for the future.